sqlserver/datadog_checks/sqlserver/statements.py

```diff
@@ -292,6 +292,7 @@ def _to_metrics_payload(self, rows):
             'timestamp': time.time() * 1000,
             'min_collection_interval': self.collection_interval,
             'tags': self.check.tags,
+            'cloud_metadata': self.check.cloud_metadata,
```

Comment:
I know we talked about adding extra logic here around if a user has set `host` to an rds instance endpoint, we should set that to be our `aws` metadata, but we can do that mapping logic on the backend